string json_desiredstate
---
bool state_reached
